### DISCUSSION TITLE
[Feature Request] VST/LV2/LADSPA plugin support

### DISCUSSION THREAD

**Discussion by hugeblank** [theRealCarneiro/pulsemeeter#224] [Ideas]
Similar to how OBS handles VST 2.x audio plugins in the form of a filter, to be able to apply *any* filter on *any* input (hardware or virtual) could be awesome. Bonus points if the implementation supports using a hotkey or command (eg: `pulsemeeter input=HW1 filter=1 disable` to disable Filter 1, the plugin, on Hardware Input 1) to enable/disable them on the fly, or through something like a Stream Deck.

**Comment by theRealCarneiro** [comment]
I'm working on something like that with ladspa, still very early stage because it's very annoying to use plugins with pulse, and it introduces very high latency

**Comment by kadisonm** [comment]
Hi, has this since been implemented or looked at? Looking to switch to Linux soon but currently am using VM.

I don't fully understand how this works but I'm using VM's ASIO to feed into Cantabile (VST host) which outputs back into VM. If Pulsemeeter could host its own VST plugins that would be amazing and way simpler.

**Comment by theRealCarneiro** [comment]
Hello, this has not been implemented yet but its one of my priorities on my todo list
